app/controllers/SeriesDetailsController.scala

```diff
@@ -0,0 +1,39 @@
+package controllers
+
+import javax.inject.{Inject, Singleton}
+import org.pac4j.core.profile.CommonProfile
+import org.pac4j.play.scala.{Security, SecurityComponents}
+import play.api.data.Form
+import play.api.data.Forms._
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, Request}
+
+@Singleton
+class SeriesDetailsController @Inject()(val controllerComponents: SecurityComponents)
+                                         extends Security[CommonProfile] with I18nSupport  {
+
+  private val secureAction = Secure("OidcClient")
+
+  val selectedSeriesForm = Form(
+    mapping(
+      "seriesId" -> number
```

Comment:
How does `number` work here, given that the mock IDs are strings like `mockedSeries1`?